client/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import "./Banner.css";
+
+const Banner = props => {
+  let classlist = "banner";
+  return <div className={classlist} style={{
+    backgroundImage: `${props.bannerType}`,
+    height: `${props.bannerHeight}`, fontSize: `${props.bannerFontSize}`, lineHeight: `${props.bannerLineHeight}`
```

Comment:
This is cool, but font size and line height should be determined by the CSS. Could we do an enum style prop for height?